source/educators/references/course_development/exercise_tools/enable_poll_OLX.rst

```diff
@@ -1,13 +1,12 @@
-Alternatively, you can use OLX to enable the poll tool.
-
 .. _Enable Poll in OLX:
 
-======================================
 Enable the Poll Tool in OLX
-======================================
+############################
 
 .. tags:: educator, reference
 
+.. note:: Alternatively, you can use OLX to enable the poll tool.
```

Comment:
I don't understand why this note was added, without a link to the alternative option. What should I do with this note?